cleansweep/volunteers/views.py

```diff
@@ -33,6 +34,18 @@ def add_volunteer(place):
     return render_template("add_volunteer.html", place=place, form=form)
 
 
+@plugin.place_view("/volunteers/download")
+def download_volunteer(place):
+    headers = ['Name', "Phone", 'Email', 'Voter ID', 'Location', 'Timestamp']
+    data = tablib.Dataset(headers=headers)
+    for m in place.get_all_members():
+        data.append([m.name, m.phone, m.email, m.voterid, m.place.name, m.created.isoformat()])
+    response = make_response(data.xls)
+    response.headers['content_type'] = 'application/vnd.ms-excel;charset=utf-8'
+    response.headers['Content-Disposition'] = "attachment; filename=export.xls"
```

Comment:
Could you please name the file as place-key + "-volunteers.xls'. So for `DL/AC001`, it'll be `DL-AC001-volunteers.xls`.